src/scrape_up/coursera/courses.py

```diff
@@ -22,7 +22,17 @@ def __scrape_page(self):
 
     def titles(self):
         """
-        Returns the titles of courses
+        Class - `Courses`
+        Example:
+        ```
+        topic = Courses("Machine Learning", 10)
+        courses = topic.titles()
+        ```
+        Returns:
+        {
+            "data": titles,
+            "message": f"Course Titles for {self.keyword}"
+        }
```

Comment:
Issue #267 was only there to add docstring in the Coursera class, which looks fine. But why are you pushing the changes you made in the finance and the Instagram class with it? Remove those changes in this PR. Only push the docstring changes in this PR.